### PR TITLE
1582 mock add plugin buttons

### PR DESCRIPTION
**Pull request by tibetsprague**
Fixes https://github.com/daostack/alchemy/issues/1582

### CODE REVIEW

File: .slugignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /test
 /.storybook
-/stories
\ No newline at end of file
+/stories
+./node_modules/3box
```
Comment (tibetsprague) [inline .slugignore]: I dont think we can do this, we need 3box to be available

File: src/components/Dao/DaoSchemesPage.tsx [hunk 2]
```diff
@@ -98,13 +140,21 @@ class DaoSchemesPage extends React.Component<IProps, null> {
...
   checkForUpdate: [],
   createObservable: (props: IExternalProps) => {
+    const arc = getArc();
     const dao = props.daoState.dao;
-    return dao.schemes({ where: { isRegistered: true } }, { fetchAllData: true, subscribe: true });
+
+    return combineLatest(
+      dao.schemes({ where: { isRegistered: true } }, { fetchAllData: true, subscribe: true }),
+      // Find the SchemeManager scheme if this dao has one
+      Scheme.search(arc, {where: { dao: dao.id, name: "SchemeRegistrar" }}).pipe(mergeMap((scheme: Array<Scheme | CompetitionScheme>): Observable<ISchemeState> => scheme[0] ? scheme[0].state() : of(null)))
```
Comment (dkent600) [inline src/components/Dao/DaoSchemesPage.tsx:155]: why is CompetitionScheme needed here?  We're looking for SchemeRegistrar.

can we document that in the code?

File: src/components/Scheme/SchemeContainer.tsx [hunk 3]
```diff
@@ -257,6 +232,8 @@ const SubscribedSchemeContainer = withSubscription({
 
     return combineLatest(
       of(schemeState),
+      // Find the SchemeManager scheme if this dao has one
+      Scheme.search(arc, {where: { dao: props.daoState.id, name: "SchemeRegistrar" }}).pipe(mergeMap((scheme: Array<Scheme | CompetitionScheme>): Observable<ISchemeState> => scheme[0] ? scheme[0].state() : of(null))),
```
Comment (dkent600) [inline src/components/Scheme/SchemeContainer.tsx:236]: why is CompetitionScheme needed here? We're looking for SchemeRegistrar.

can we document that in the code?

File: src/components/Dao/DaoSchemesPage.tsx (hunk 2 above)
Comment (tibetsprague) [reply src/components/Dao/DaoSchemesPage.tsx:155]: thats the type returned by Scheme.search so thats why we need to use it

File: .slugignore (hunk 1 above)
Comment (orenyodfat) [reply .slugignore]: this was by mistake and was reverted

File: src/components/Scheme/SchemeContainer.tsx (hunk 3 above)
Comment (tibetsprague) [reply src/components/Scheme/SchemeContainer.tsx:236]: its not, its just the required type

**Comment by orenyodfat** [comment]
while trying to propose via plugin manager I can only remove schemes...
trying to edit/add gives me 
Screen Shot 2020-04-20 at 13 44 51
I guess we do not want to totally remove this functionality.

**Comment by orenyodfat** [comment]
@EzraWeller we are missing some copy and design

**Comment by tibetsprague** [comment]
@EzraWeller updated the copy and styling. can you check again?

**Comment by EzraWeller** [comment]
Looking good, thanks @tibetsprague !

The only issue that I see is that the "Edit Plugin" modal is displaying the copy for the "Add Plugin" modal: 
Screen Shot 2020-04-30 at 12 26 33 PM

**Comment by tibetsprague** [comment]
oops @EzraWeller what should it show? should it just show the form always?

**Comment by EzraWeller** [comment]
Check the mocks from the #1582 ! It shows some different copy and doesn't have the list of plugins Let me know if you can't find what I'm talking about.

**Comment by dkent600** [comment]
@EzraWeller Just committed hopefully a fix for the "edit" case.

**Comment by EzraWeller** [comment]
Small typo here, then approved (should be "You'll" with a single apostrophe):
Screen Shot 2020-05-04 at 11 12 50 AM

**Comment by dkent600** [comment]
@EzraWeller 

> Small typo here

Fixed

**Pull request merged**